### DISCUSSION TITLE
How does one become a collaborator for Netlify CMS?

### DISCUSSION THREAD

**Discussion by bytrangle** [decaporg/decap-cms#5847] [Q&A] [answered]
Hopefully this is the right place to ask. Notice that the pull requests have been piling up recently. Some are stalled while still in the review process. So I'd like to help out if I can.

**Comment by erezrokah** [comment, marked as answer]
Hi @bytrangle, this is the right place and thank you for raising it!
I came back from vacation today (hence the pile up), so I'm going over the PRs as much as possible.

We are in the process of discussing this internally and trying to come up with a standard way to do this for all of our open source repos, so I'll update once I have some answers.

Regardless, if you have feedback on any open PR, please do comment on it.

**Reply by bytrangle** [reply]
Thank you for the speedy response, as always. When I raised my question, I didn't mean to criticize your review speed nor push you to review faster. The number of open source projects I've contributed is still a single digit, but Netlify CMS is still the best experience for me.

Regards.
